### PR TITLE
camera-streaming-daemon: Add Camera Streaming Daemon recipe

### PR DESCRIPTION
**Pull request by otaviobp**
Camera Streaming Daemon is a daemon to discover, publish and
stream videos from a drone.

This recipe fixes issue #49 
This still depends on fixing and merging the pull request https://github.com/01org/camera-streaming-daemon/pull/30 from camera-streaming-daemon.

### CODE REVIEW

File: recipes-support/camera-streaming-daemon/camera-streaming-daemon_0.1.bb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+S = "${WORKDIR}/git"
+
+inherit autotools pkgconfig
+
+do_install_append () {
+	install -d ${D}${sysconfdir}/init.d
+	install -m 0755 ${WORKDIR}/camera-streaming-daemon.sh ${D}${sysconfdir}/init.d
+
+    install -d ${D}${sysconfdir}/camera-streaming-daemon/
```
Comment (zehortigoza) [inline recipes-support/camera-streaming-daemon/camera-streaming-daemon_0.1.bb]: nitpick: code style

File: recipes-support/camera-streaming-daemon/camera-streaming-daemon_0.1.bb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    install -m 0555 ${WORKDIR}/main.conf ${D}${sysconfdir}/camera-streaming-daemon/
+
+	install -d ${D}${sysconfdir}/rc1.d
+	ln -sf ../init.d/camera-streaming-daemon.sh ${D}${sysconfdir}/rc1.d/S71mavlink-routerd
```
Comment (zehortigoza) [inline recipes-support/camera-streaming-daemon/camera-streaming-daemon_0.1.bb]: **S71mavlink-routerd**

Comment (otaviobp) [inline recipes-support/camera-streaming-daemon/camera-streaming-daemon_0.1.bb]: ops. my bad

**Comment by otaviobp** [comment]
Branch updated and tested with camera-streaming-daemon with config support

**Comment by lucasdemarchi** [comment]
This has been applied.

**Pull request closed without merging**